pkg/controllermanager/controller/bastion/add_test.go

```diff
@@ -189,3 +209,60 @@ var _ = Describe("Add", func() {
 		})
 	})
 })
+
+// TODO: remove this again once the controller-runtime fake client supports field selectors
```

Comment:
Is there already a PR for it or can you open one with this code? :)

Reply:
Unfortunately, there is no PR for this upstream yet, AFAIK.
This will probably involve some design discussion, so expect some delay. But I will try my best :)

Reply:
😄 Okay, thank you!